src/middlewared/middlewared/plugins/device_/device_info_linux.py

```diff
@@ -14,9 +14,10 @@
 from middlewared.utils import osc
 
 RE_DISK_SERIAL = re.compile(r'Unit serial number:\s*(.*)')
-RE_NVME_PRIVATE_NAMESPACE = re.compile(r'nvme[0-9]+c')
 RE_SERIAL = re.compile(r'state.*=\s*(\w*).*io (.*)-(\w*)\n.*', re.S | re.A)
 RE_UART_TYPE = re.compile(r'is a\s*(\w+)')
+RE_NVME_PRIV = re.compile(r'nvme[0-9]+c')
+IGNORE = ('sr', 'md', 'dm-', 'loop', 'zd')
```

Comment:
We have the exact same set in `disk_events.py` and almost the same set in `enclosure.py`, can we use a common variable?